src/functionalTest/resources/envelopes/new-envelope-create-case-with-evidence.json

```diff
@@ -0,0 +1,45 @@
+{
+  "id": "{ENVELOPE_ID}",
+  "case_ref": "",
+  "po_box": "BULKSCAN PO BOX",
+  "jurisdiction": "BULKSCAN",
+  "container": "bulkscan",
+  "formType": "B123",
+  "classification": "NEW_APPLICATION",
+  "zip_file_name": "zip-file-test.zip",
+  "delivery_date": "1970-01-01T00:00:00.000Z",
+  "opening_date": "1970-01-01T00:00:00.000Z",
+  "documents": [
+    {
+      "file_name": "certificate1.pdf",
+      "control_number": "154565768",
+      "type": "other",
+      "subtype": null,
+      "scanned_at": "2018-01-01T10:30:00.000Z",
+      "uuid": "ee69aee8-1a33-40dd-9af9-d90da1b104ba"
+    }
+  ],
+  "ocr_data": [
+    {
+      "metadata_field_name": "first_name",
+      "metadata_field_value": "value1"
+    },
+    {
+      "metadata_field_name": "last_name",
+      "metadata_field_value": "value2"
+    },
+    {
+      "metadata_field_name": "email",
+      "metadata_field_value": "hello@test.com"
+    }
+  ],
+  "ocr_data_validation_warnings": [
+    "warning 1",
+    "warning 2"
+  ],
+  "payments": [
```

Comment:
Should we exclude payments in the request?

Reply:
This is the content of a queue message, so I guess it doesn't matter if we include them or not. Might just be clearer if they're excluded.

Reply:
Done